api/dp-openapi.yaml

```diff
@@ -1290,6 +1290,49 @@ paths:
               schema:
                 $ref: '#/components/schemas/round_stats'
 
+  /documents:
+    get:
+      tags:
+      - documents
+      description: Gets an array of document names, if lang_code is given then only those available in that language
+      parameters:
+      - name: lang_code
+        in: query
+        description: language code
+        schema:
+          type: string
+      responses:
+        200:
+          description: Array of document names
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+
+  /documents/{document_name}:
```

Comment:
Perhaps it should have been obvious to me, and will be to those who actually use the API to pull the documentation, but I was unsure just what string to use to pull the URL to the documentation when I first started testing. Would it be a good idea to document that the document names should be drawn from the list of documents in the results from the `/documents` query?

Everything else looks good.
---

Specifying a 2-letter language code worked just fine for `/documents`, inasmuch as the list that was returned was limited to only the docs available in that language.

If I had my language pref set for French for `/documents/{document_name}`, `curl` correctly gave me the link to the French version of the proofreading guidelines 🎉 . However, when testing in Swagger, `/documents/{document_name}` does not return the language result--I assume because Swagger doesn't have access to my prefs on TEST.